pipeline_tools/get_analysis_metadata.py

```diff
@@ -18,19 +28,81 @@ def get_workflow_id(analysis_output_path):
     url = analysis_output_path
     calls = url.split('/call-')
     workflow_id = calls[1].split('/')[-1]
+    print('Got analysis workflow UUID: {0}'.format(workflow_id))
     with open('workflow_id.txt', 'w') as f:
         f.write(workflow_id)
     return workflow_id
 
 
+def get_adapter_workflow_id(analysis_output_path):
+    """Parse the adapter workflow id from one of its analysis workflow output paths.
+
+    Args:
+        analysis_output_path (str): Path to workflow output file.
+
+    Returns:
+        workflow_id (str): String giving Cromwell UUID of the adapter workflow.
+    """
+    url = analysis_output_path
+    calls = url.split('/call-')
+    adapter_workflow_id = calls[0].split('/')[-1]
+    print('Got adapter workflow UUID: {0}'.format(adapter_workflow_id))
+    with open('adapter_workflow_id.txt', 'w') as f:
+        f.write(adapter_workflow_id)
+    return adapter_workflow_id
+
+
+def get_adapter_workflow_version(runtime_environment,
```

Comment:
I don't think we should be making a call to Cromwell for this. It adds another external dependency and possible point of failure that I'd rather not have. It would simplify things if we hard code a version parameter in each adapter WDL or analysis WDL and pass it to the task that needs it, e.g. `String pipeline_version = v1.2.3`.

Reply:
Per an offline discussion with @dshiga and also @jishuxu  I will make a PR in Skylab for this. changing both ss2 and Optimus. I agree that we cannot really rely on the workflow labels for everything.